_aggregations/metric/weighted-average.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: Weighted Average
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 150
+redirect_from:
+  - /query-dsl/aggregations/metric/weighted-average/
+---
+
+# Weighted average aggregations
+
+The `weighted average` metric is a multi-value metric aggregations that returns the weighted average value. Weighted average can be used when calculating grades for students, where various assignments and exams make-up the final grade of 100%. 
+
+
+A weighted average has two fields, a value and a weight.
+
+* `value`: value that the weight is applied to
+  * `missing`: (optional) to apply a default value for any missing/null value fields
+* `weight`: weight that the value is applied for
+  * `missing`: (optional) to apply a default weight for any missing/null weight fields
+
+Every value field is applied with the respective value in the weight field.
+
+![Weighted average formula]({{site.url}}{{site.baseurl}}/images/aggregations/metric/weighted-average.png)
```

Comment:
I'm a bit worried that this causes some inconsistency with size and font from existing formulas like https://github.com/opensearch-project/documentation-website/pull/6808/files